airbyte-config/config-models/src/main/java/io/airbyte/config/Configs.java

```diff
@@ -584,6 +584,11 @@ public interface Configs {
    */
```

Comment:
Throughout this diff, the details around things like environment variable configuration and worker registration will change considerably when we implement this for real. I kept it very naive for the spike just to get a proof of concept working